src/hooks/useSovereignVenue.ts

```diff
@@ -42,6 +42,8 @@ export const useSovereignVenue: ReactHook<
     isLoading: isSovereignVenueLoading,
   } = useSelector(sovereignVenueSelector, isEqual);
 
+  console.log("hook", sovereignVenue);
+
```

Comment:
```suggestion
```

Reply:
Removed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0e1ea16c81870c898660c5e5b60aa9a143b57c80